package/libpiksi/libpiksi/include/libpiksi/minGlue.h

```diff
@@ -9,6 +9,8 @@
  */
 
 /* map required file I/O types and functions to the standard C library */
+#ifndef LIBPIKSI_MINGLUE_H

```

Comment:
Let's fix this filename to follow our naming scheme (and include "ini" in the name so it's clear that it's part of minIni): `minGlue.h` -> `min_ini_glue.h`